docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 ## April 2024
 
+**25 April 2024**
+
+Frontend version: 1.0.0-20240419.1106 | Backend version: 1.99.1-20240422.1153
+
+| Type  | Change| Description |
+|-------|-------|-------------|
+| **Change** | Automation API optimisation | `get/list users` will exclude `seedStatus` and `seedDevices` by default. To include these, pass the query param `seedInfo=true`. Similarly, `list application role assignments` will exclude TechBiz group assignments by default, pass `onTechBiz=true` if required. Affected APIs include: `get user info`, `list user accounts`, `list user accounts by namespace`, `list product accounts`, `list product accounts by namespace`. |
```

Comment:
can separate out`list application role assignments` into its own change since they are 2 separate changes (The 'affected APIs' goes with the `get/list users` change). also I made a mistake, should be `includeOnTechBiz=true`, instead of `onTechBiz=true`.